backend/service/k8s/job_test.go

```diff
@@ -126,6 +127,36 @@ func TestDeleteJob(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+var batchJob = `
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test-job
+  labels:
+    environment: staging
+    facet-type: batch
+  namespace: testing-namespace
+`
+
+func TestCreateJob(t *testing.T) {
+	cs := testJobClientset()
+
+	s := &svc{
+		manager: &managerImpl{
+			clientsets: map[string]*ctxClientsetImpl{"testing-clientset": &ctxClientsetImpl{
+				Interface: cs,
+				namespace: "testing-namespace",
+				cluster:   "testing-cluster",
+			}},
+		},
+	}
+
+	value := structpb.NewStringValue(batchJob)
+	job, err := s.CreateJob(context.Background(), "testing-clientset", "testing-cluster", "testing-namespace", value)
```

Comment:
is there a common error case we can test as well?

Reply:
Added a case where the job Config passed in is invalid and it returns an error